lib/resty/memcached/shdict.lua

```diff
@@ -256,7 +256,9 @@ function _M.gen_memc_methods (opts)
                 lock:unlock()
             end
 
-            error_log(ctx, "failed to init ", tag, ": ", err)
+            error_log(ctx, "failed to init ", tag, ": ", err,
+                      ', the dst ip is: ', memc_host,
+                      ', the dst port is: ', memc_port)
```

Comment:
Can we use more compact error message text format like `, dst: <ip>:<port>`?

Reply:
fixed by https://github.com/openresty/lua-resty-memcached-shdict/pull/6